README.md

```diff
@@ -334,3 +334,13 @@ public partial class InvoicePage : Json, IBound<Invoice>
 ```
 
 Method marked with this attribute should be public and static, return `Permission` and accept context as its argument. Whatever permission it returns is then checked to see if the user can access the page.
+
+# How to start
+Minimum reasonable configuration to start playing around with this library is to use sample [apps provided by Starcounter](https://github.com/StarcounterApps):
```

Comment:
It's not a minimal configuration, it's a quick start configuration. There's a difference. Minimal would be just SignIn or equivalent